### PR TITLE
Doc: Mention FORCE_REGENERATE var

### PR DESCRIPTION
**Pull request by SmartManoj**
**Give a brief summary of what the PR does, explaining any non-trivial design decisions**
Renamed SKIP_TEST to FORCE_REGENERATE  and mentioned in doc.

### CODE REVIEW

**Review by li-boxuan — COMMENTED** [review]
I appreciate the effort to polish the doc! I'd be happy to approve it if you don't change the env name.

**Comment by SmartManoj** [comment]
It is skipping the test in <ins>**Step 1** </ins> only. The remaining tests will be run. So, it is ambiguous.
The variable is aligned with `FORCE_USE_LLM`

File: tests/integration/README.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ to cover the cost, ask one of the maintainers to regenerate for you. Before aski
 please try running the script first *without* setting `LLM_API_KEY`.
 Chance is, the test could be fixed after step 2.
 
+## Regenerate Integration Tests without testing first
+
+If you want to regenerate all tests without running the existing tests first, you can run:
```
Comment (li-boxuan) [inline tests/integration/README.md:105]:
```suggestion
If you want to regenerate all prompts and/or responses without running the existing tests first, you can run:
```

**Comment by li-boxuan** [comment]
> It is skipping the test in **Step 1** only. The remaining tests will be run. So, it is ambiguous. The variable is aligned with `FORCE_USE_LLM`

That's a fair point. Approved.

**Review by li-boxuan — APPROVED** [review]

**Pull request merged**